src/resources/js/views/month-mobile-events.js

```diff
@@ -0,0 +1,193 @@
+ *
+ * @return {void}
+ */
+( function( $, obj ) {
+	'use strict';
+	var $document = $( document );
+
+	/**
+	 * Selectors used for configuration and setup
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.selectors = {
+		calendar: '.tribe-events-calendar-month',
+		calendarDay: '.tribe-events-calendar-month__day-cell--mobile',
+		calendarDaySelected: '.tribe-events-calendar-month__day-cell--selected',
+		mobileEventsDay: '.tribe-events-calendar-month-mobile-events__mobile-day',
+	};
+
+	/**
+	 * Closes all mobile events
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $container jQuery object of view container
+	 *
+	 * @return {void}
+	 */
+	obj.closeAllEvents = function( $container ) {
+		$container.find( obj.selectors.calendarDay ).each( function( index, header ) {
+			var $header = $( header );
+			var contentId = $header.attr( 'aria-controls' );
+
+			/**
+			 * Define empty jQuery object in the case contentId is false or undefined
+			 * so that we don't get selectors like #false or #undefined.
+			 * Also only perform accordion actions if header has aria-controls attribute.
+			 */
+			var $content = $( '' );
+			if ( contentId ) {
+				$content = $container.find( '#' + contentId );
+				tribe.events.views.accordion.closeAccordion( $header, $content );
+			}
+
+			obj.closeMobileEvents( $header, $content );
+		} );
+	};
+
+	/**
+	 * Opens mobile events
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $header jQuery object of mobile day button
+	 * @param {jQuery} $content jQuery object of mobile events container
+	 *
+	 * @return {void}
+	 */
+	obj.openMobileEvents = function( $header, $content ) {
+		// only perform accordion actions if $header has aria-controls attribute.
+		var contentId = $header.attr( 'aria-controls' );
+		if ( contentId ) {
+			tribe.events.views.accordion.openAccordion( $header, $content );
+		}
+
+		$header.addClass( obj.selectors.calendarDaySelected.className() );
+		$content.parent().css( 'display', 'block' );
+	};
+
+	/**
+	 * Closes mobile events
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $header jQuery object of mobile day button
+	 * @param {jQuery} $content jQuery object of mobile events container
+	 *
+	 * @return {void}
+	 */
+	obj.closeMobileEvents = function( $header, $content ) {
+		// only perform accordion actions if $header has aria-controls attribute.
+		var contentId = $header.attr( 'aria-controls' );
+		if ( contentId ) {
+			tribe.events.views.accordion.closeAccordion( $header, $content );
+		}
+
+		$header.removeClass( obj.selectors.calendarDaySelected.className() );
+		$content.parent().css( 'display', '' );
+	};
+
+	/**
+	 * Toggles mobile events on mobile day click
+	 *
+	 * @since TBD
+	 *
+	 * @param {Event} event event object of click event
+	 *
+	 * @return {void}
+	 */
+	obj.toggleMobileEvents = function( event ) {
+		var $container = event.data.container;
+		var $header = $( event.data.target );
+		var contentId = $header.attr( 'aria-controls' );
```

Comment:
Same as the above, would love to see this being selector, not an ID only kinda thing.